lib/rof/get_from_fedora.rb

```
@@ -0,0 +1,161 @@
+require 'json'
+require 'rexml/document'
+require 'rdf/ntriples'
+require 'rdf/rdfxml'
+require 'rubydora'
+
+module ROF
+  # connect to fedora and fetch object
+  # returns array of fedora attributes or nil
+
+  def self.GetFromFedora(pid, fedora, config)
+    @fedora_info = {}
+
+    # Try to connect to fedora, and search for the desired item
+    # If either of these actions fail, handle it.
+    begin
+      fedora = Rubydora.connect(fedora)
+      doc = fedora.find(pid)
+    rescue StandardError => e
+      puts "Error: #{e}"
+      return @fedora_info
```

Why is this exception being swallowed? There is an error being output to standard out, but should this exit with a non-zero?